main.py

```diff
@@ -102,6 +105,11 @@ def search_key():
                     start = True
                     break
 
+		elif exitRect.collidepoint(event.pos):
+                    exit = True
+		    pygame.quit()

```

Comment:
My editor sees uneven spacing with tabs, spaces etc. Not sure if it's only something I see but a reminder to use spaces only, please check the general note on the Gitter chat.